### PR TITLE
[DOCS] Getting Started with Kyuubi and Cloudera Hue

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #425](https://github.com/yaooqinn/kyuubi/pull/425)    Target Issue Doc [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

image

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/quick_start/quick_start_with_hue.md [hunk 1]
```diff
@@ -4,11 +4,115 @@
...
+
+```
+docker run --rm gethue/hue:latest cat /usr/share/hue/desktop/conf/hue.ini > hue.ini
+```
+
+2. Modify the `hue.ini`
+
+```ini
+[beeswax]
+  # Kyuubi 1.1.x support thrift version from 1 to 9
```
Comment (yaooqinn) [inline docs/quick_start/quick_start_with_hue.md:50]: 10？

File: conf/kyuubi-defaults.conf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#
```
Comment (yaooqinn) [inline conf/kyuubi-defaults.conf:1]: what is this for?

**Comment by yaooqinn** [comment]
Looks awesome!

File: docs/quick_start/quick_start_with_hue.md (hunk 1 above)
Comment (pan3793) [reply docs/quick_start/quick_start_with_hue.md]: Yes, fixed

File: conf/kyuubi-defaults.conf (hunk 2 above)
Comment (pan3793) [reply conf/kyuubi-defaults.conf]: mis commit, reverted

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.2.0

**Comment by ulysses-you** [comment]
@yaooqinn @pan3793  seems you have not mergerd successfully .. so let me merging to master again.
